stories/button.stories.js

```diff
@@ -184,3 +185,45 @@ Elevated.args = { variant: 'elevated' }
 
 export const Filled = Template.bind({})
 Filled.args = { variant: 'filled' }
+
+const Template1 = (a) => (
+  <>
+    <div>
+      <p className='text-primary-700 mb-3'>CUSTOM STYLING:</p>
+      <Button variant={a.variant} classes={a.classes}>Custom Button</Button>
+    </div>
+  </>
+)
+
+const customStyleClasses = makeStyles({
```

Comment:
This should probably be defined in the component itself rather than the story. This will be the default look for Paymi buttons

Also use `rem` instead of `px`

Reply:
The buttons have a variant, I think we should make a variant for the paymi buttons otherwise we're going to be redefining them each time in actual use. What do you think @kc-leung?

Reply:
@Hamaad102 all the Paymi styling will be done in Paymi repo itself.  We agreed to keep lumen-labs as it was initially designed for, to answer EQ Works specific products. Beside that, we are now allowing for customized styling, but all of that customized styling will be done in the repos that need it not in lumen-labs.

Reply:
mmm lets bring this up to the thread

Reply:
> This should probably be defined in the component itself rather than the story. This will be the default look for Paymi buttons
> 
> Also use `rem` instead of `px`

This is just an example of customized styling, it doesn't matter what we use. I just used the Paymi colours and styling to show what it can be done.